.github/workflows/docker-publish.yml

```diff
@@ -0,0 +1,76 @@
+name: Publish to Docker registry
+
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+on:
+  push:
+    branches: ['master']
+  workflow_dispatch:
+  repository_dispatch:
+
+env:
+  # Use docker.io for Docker Hub if empty
+  REGISTRY: ghcr.io
+  # github.repository as <account>/<repo>
+  IMAGE_NAME: tonyrl/docker-notify
```

Comment:
These are the wrong values for this repo. Can you remove this file please? I will set up a new docker build and publish workflow later.

Reply:
Yes, removed this file.